docker-compose.yml

```diff
@@ -74,8 +74,10 @@ services:
     <<: *base_service
     profiles: ["comfy"]
     build: ./services/comfy/
-    image: sd-comfy:1
+    image: sd-comfy:2
     tty: true
+    volumes:
+      - *v2
```

Comment:
This is already mounted in the `x-base-service`, so there is no need to change the mounts here.

Reply:
That's the issue though. I know it's present, but the save image node won't save the images to the hosts output directory unless it is present.

Reply:
What is the output folder of the save image node? and does the `output/comfy` exist on your machine?